homeassistant/components/light/xiaomi_aqara.py

```diff
@@ -64,15 +64,15 @@ def parse_data(self, data, raw_data):
         brightness = rgba[0]
         rgb = rgba[1:]
 
-        self._brightness = int(255 * brightness / 100)
+        self._brightness = brightness
         self._hs = color_util.color_RGB_to_hs(*rgb)
         self._state = True
         return True
 
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
```

Comment:
Change the doc string too

Reply:
Hmm, the property `self.brightness` is actually 0..255.

Reply:
Ahh, sorry